dvc/remote/gdrive/__init__.py

```diff
@@ -46,6 +47,10 @@ def _wrap_pydrive_retriable(call):
 )
 
 
+def get_tmp_filepath():
```

Comment:
I think we have helpers to do this cc @efiop ?

Reply:
Yes. @MaxRis Take a look at dvc/utils/__init__.py: https://github.com/iterative/dvc/blob/master/dvc/utils/__init__.py#L268

Reply:
thanks! was not able to find something relevant